core/components/atoms/icon/icons.json

```diff
@@ -68,18 +68,18 @@
       "width": 20,
       "height": 20
     },
+    "chevron-right-fill": {
+      "paths": ["M7.903 2L6 3.88 12.18 10 6 16.12 7.903 18 16 10z"],
+      "width": 20,
+      "height": 20
+    },
     "chevron-right": {
       "paths": [
         "M13.76 9.659a.524.524 0 0 1-.152.394c-.024.024-.06.017-.087.035L6.903 16.16a.53.53 0 0 1-.747-.749l6.273-5.754-6.274-5.755a.528.528 0 1 1 .748-.747l6.616 6.068c.028.02.065.013.09.038.11.109.156.254.15.398z"
       ],
       "width": 20,
       "height": 20
     },
-    "chevron-right-fill": {
-      "paths": ["M7.903 2L6 3.88 12.18 10 6 16.12 7.903 18 16 10z"],
-      "width": 20,
-      "height": 20
-    },
```

Comment:
:thinking: what's even happening here with `lint-staged`?

Reply:
Weird 😅

Reply:
Ran into this myself. I tried a few different things to get the right stuff committed but gave up. 😞